cpp/examples/interop/CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ project(
 
 include(../fetch_dependencies.cmake)
 
+include(rapids-cmake)
+rapids_cmake_build_type("Release")
+
 # The Arrow CMake is currently broken if the build type is not set
 set(CMAKE_BUILD_TYPE Release)
```

Comment:
Is this still needed?

Reply:
I'll test locally and see. I'm not sure.

Reply:
Oh I guess I filed a fix for this in https://github.com/apache/arrow/pull/43794/. I think it made it into Arrow 18, so we can't drop this until we switch to requiring that version ([we currently use 16.1](https://github.com/rapidsai/cudf/blob/branch-25.02/cpp/cmake/thirdparty/get_arrow.cmake#L350)). We could upgrade any time we wanted though since the library used for these builds is no longer tied to libcudf itself in any way. I'd be fine with a follow-up PR to upgrade that value and remove this setting.